watertap/unit_models/tests/test_uv_aop.py

```diff
@@ -741,8 +732,242 @@ def test_costing(self, UV_frame):
         assert_optimal_termination(results)
 
         # Check solutions
-        assert pytest.approx(1544038, rel=1e-5) == value(m.fs.unit.costing.capital_cost)
+        assert pytest.approx(1767152, rel=1e-5) == value(m.fs.unit.costing.capital_cost)
         assert pytest.approx(90400.7, rel=1e-5) == value(
             m.fs.unit.costing.fixed_operating_cost
         )
-        assert pytest.approx(0.0192355, rel=1e-5) == value(m.fs.costing.LCOW)
+        assert pytest.approx(0.02023034, rel=1e-5) == value(m.fs.costing.LCOW)
+
+
+class TestUV_detailed:
+    @pytest.fixture(scope="class")
+    def UV_frame(self):
+        m = ConcreteModel()
+        m.fs = FlowsheetBlock(default={"dynamic": False})
+
+        m.fs.properties = props.NDMAParameterBlock()
+
+        m.fs.unit = Ultraviolet0D(
+            default={
+                "property_package": m.fs.properties,
+                "uv_dose_type": "calculated",
```

Comment:
May want to test
`import UVDoseType`
and
```suggestion
                "uv_dose_type": UVDoseType.calculated,
```